.github/workflows/release.yml

```diff
@@ -7,26 +7,52 @@ on:
 
 jobs:
   build_addon:
+    runs-on: windows-2019
+    steps:
+    - name: Set VERSION env
+      run: echo VERSION=${GITHUB_REF:11} >> $GITHUB_ENV
+
+    - uses: actions/checkout@v2
+
+    - name: Pull Arma3Tools
+      run: |
+        echo ${{ secrets.CR_PAT }} | docker login -u ${{ secrets.CR_USER }} --password-stdin ghcr.io
+        docker pull ghcr.io/armaforces/arma3tools:latest
+
+    - name: Build using HEMTT
+      run: |
+        docker run -v "$(pwd):c:/workdir" -e CI=$True ghcr.io/armaforces/arma3tools:latest hemtt build --release --ci --time
+
+    - uses: actions/upload-artifact@v2
+      with:
+        name: releases
+        path: releases/
```

Comment:
In `arma.yml` you have:
```yml
    - uses: actions/upload-artifact@v2
      with:
        name: armaforces_mods
        path: releases/*.zip
```
The `*.zip` is not needed here?

Reply:
It's not, we need zip file to attach to GitHub release but raw files for Steam Workshop upload.